src/pages/settings/Wallet/WalletPage/WalletPage.js

```diff
@@ -0,0 +1,532 @@
+import React, {useCallback, useEffect, useState, useRef} from 'react';
+import {ActivityIndicator, View, InteractionManager, ScrollView} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import PaymentMethodList from '../PaymentMethodList';
+import ROUTES from '../../../../ROUTES';
+import HeaderWithBackButton from '../../../../components/HeaderWithBackButton';
+import ScreenWrapper from '../../../../components/ScreenWrapper';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import styles from '../../../../styles/styles';
+import compose from '../../../../libs/compose';
+import * as BankAccounts from '../../../../libs/actions/BankAccounts';
+import Popover from '../../../../components/Popover';
+import MenuItem from '../../../../components/MenuItem';
+import * as PaymentMethods from '../../../../libs/actions/PaymentMethods';
+import getClickedTargetLocation from '../../../../libs/getClickedTargetLocation';
+import CurrentWalletBalance from '../../../../components/CurrentWalletBalance';
+import ONYXKEYS from '../../../../ONYXKEYS';
+import Permissions from '../../../../libs/Permissions';
+import AddPaymentMethodMenu from '../../../../components/AddPaymentMethodMenu';
+import CONST from '../../../../CONST';
+import * as Expensicons from '../../../../components/Icon/Expensicons';
+import KYCWall from '../../../../components/KYCWall';
+import {propTypes, defaultProps} from './walletPagePropTypes';
+import {withNetwork} from '../../../../components/OnyxProvider';
+import * as PaymentUtils from '../../../../libs/PaymentUtils';
+import OfflineWithFeedback from '../../../../components/OfflineWithFeedback';
+import ConfirmContent from '../../../../components/ConfirmContent';
+import Button from '../../../../components/Button';
+import themeColors from '../../../../styles/themes/default';
+import variables from '../../../../styles/variables';
+import useLocalize from '../../../../hooks/useLocalize';
+import useWindowDimensions from '../../../../hooks/useWindowDimensions';
+import WalletEmptyState from '../WalletEmptyState';
+import * as Illustrations from '../../../../components/Icon/Illustrations';
+import WalletSection from '../WalletSection';
+
+function BaseWalletPage({bankAccountList, betas, cardList, fundList, isLoadingPaymentMethods, network, payPalMeData, shouldListenForResize, userWallet, walletTerms}) {
```

Comment:
This is replacing the `BaseWalletPage` ? Maybe can we delete that file and this can show up as a rename instead of a new file? 🤔

Reply:
Oh yeah, shouldn't the file name match the component name here?

Reply:
@grgia @marcaaron 

Should this new version of the wallet replace the old one when this PR is merged or should we preserve both versions for testing purposes?